src/page-templates/topic-landing/topic-landing.tsx

```diff
@@ -0,0 +1,171 @@
+            <div
+                sx={{
+                    paddingBottom: 'inc160',
+                    px: ['inc40', null, 'inc50', 'inc70'],
+                    paddingTop: ['inc40', null, 'inc50', 'inc70'],
+                }}
+            >
+                <GridLayout
+                    sx={{
+                        rowGap: ['inc90', null, null, 'inc130'],
+                    }}
+                >
+                    <div sx={sideNavStyles(mainGridDesktopRowsCount)}>
+                        <SideNav currentUrl="#" items={tertiaryNavItems} />
+                    </div>
+                    {(variant !== 'light' || subTopics.length > 0) && (
+                        <>
+                            {subTopics.length > 0 && (
+                                <TopicCardsContainer
+                                    topics={subTopicsWithHrefs}
+                                    title={`${name} Topics`}
+                                />
+                            )}
+                            {variant !== 'light' && (
+                                <FeaturedCardSection content={featured} />
+                            )}
+                            {variant === 'heavy' &&
+                                contentRows.map(contentRow => {
+                                    const contentType = contentRow[0].category;
+                                    const direction =
+                                        contentType === 'Podcast'
+                                            ? 'column'
+                                            : 'row';
+                                    const contentTypeSlug =
+                                        pillCategoryToSlug.get(contentType);
+                                    return (
+                                        <CardSection
+                                            key={contentType}
+                                            content={contentRow}
+                                            title={`${contentRow[0].category}s`}
+                                            direction={direction}
+                                            allHref={`${slug}/${contentTypeSlug}`}
+                                        />
+                                    );
+                                })}
+                        </>
+                    )}
+                    <Search
+                        title={`All ${name} Content`}
+                        slug={slug}
+                        sx={{
+                            gridColumn: [
+                                'span 6',
+                                null,
+                                'span 8',
+                                'span 12',
+                                '4 / span 9',
+                            ],
+                        }}
+                    />
+
+                    {variant === 'light' && relatedTopics.length > 0 && (
```

Comment:
Just a recommendation to shorthand this but still fine as is 
`{variant === 'light' && !!relatedTopics.length && (`